s3stream/src/main/java/com/automq/stream/s3/objects/StreamObject.java

```diff
@@ -18,7 +18,24 @@ public class StreamObject {
     private long streamId;
     private long startOffset;
     private long endOffset;
-    private int attributes = ObjectAttributes.UNSET.attributes();
+    private int attributes;
+
+    public StreamObject() {
+        this(0L, 0L, 0L, 0L, 0L);
+    }
+
+    public StreamObject(long objectId, long objectSize, long streamId, long startOffset, long endOffset) {
```

Comment:
The test constructor should only add in test code. The StreamObject should keep the full fields constructor to avoid mistaking init.